packages/react-dom/src/client/ReactDOMFiberInput.js

```diff
@@ -18,8 +18,10 @@ import ReactControlledValuePropTypes from '../shared/ReactControlledValuePropTyp
 import * as inputValueTracking from './inputValueTracking';
 
 type InputWithWrapperState = HTMLInputElement & {
+  value: string,
+  defaultValue: string,
```

Comment:
are these needed, HTMLInputElement should already contain these properties i think yeah?

Reply:
Huh. Flow was yelling at me, but not anymore. Thanks!

d8925a2